pkg/osvscanner/vulnerability_result.go

```diff
@@ -73,7 +73,12 @@ func buildVulnerabilityResults(
 				for j, license := range entry.License.Override {
 					overrideLicenses[j] = models.License(license)
 				}
-				r.Infof("overriding license for package %s/%s/%s with %s\n", pkg.Package.Ecosystem, pkg.Package.Name, pkg.Package.Version, strings.Join(entry.License.Override, ","))
+				// reaching here without any override licenses means that the license should be ignored completely
+				if len(overrideLicenses) == 0 {
+					r.Infof("ignoring license for package %s/%s/%s\n", pkg.Package.Ecosystem, pkg.Package.Name, pkg.Package.Version)
+				} else {
+					r.Infof("overriding license for package %s/%s/%s with %s\n", pkg.Package.Ecosystem, pkg.Package.Name, pkg.Package.Version, strings.Join(entry.License.Override, ","))
```

Comment:
Upper case?

Reply:
I think that would be best changed in a dedicated pull request since it'll change unrelated test snapshots - I can do that if you want, and then once it's landed it'll make sense to change the `ignoring` to be upper case here

Reply:
Let's keep this as it is for now.